### PR TITLE
Media query

### PR DESCRIPTION
**Pull request by PrinceBadru**
## Desktop view.
  -add media query to css.
  -add desktop background images to the files .
  -Adjust positioning using flexbox and css grid

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,38 +188,68 @@ <h2>About me</h2>
...
+                <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""></a></li>
+                <li><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/medium.jpg" alt=""><a></li>
+                <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
+            </ul> 
+            
+
+           
+        </footer>
+
```
Comment (Olamarx) [inline index.html:249]: - [ ] You will notice there is a thin grey line before the social icons. Kindly add it.
- [ ] You will notice there is no mail address under the icon, kindly display it none for desktop version.
## Figma
image

File: index.html [hunk 2]
```diff
@@ -167,38 +188,68 @@ <h2>About me</h2>
...
+                        
+                        <li >
+                            <label for="name">
+                                <input type="text" id="name"  name="user_name" placeholder="Full name" maxlength="30" required>
+                            </label>
+                        </li>
+                        <li>
+                            <label for="mail">
+                                <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                            </label>
```
Comment (Olamarx) [inline index.html:215]: - [ ] Kindly make the mobile version remain as it should be.

## Figma 
image

## Your Design
image

File: index.html [hunk 3]
```diff
@@ -9,33 +9,43 @@
...
+     </div>
+
+
     </header>
     <br>
     <section class="second">
-      <h1>  Hey there. I’m Mariane <span class="green" >I’m a software developer</span></h1>
-    <p>
+      <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer</span></h1>
+    <p class="second-text">
```
Comment (Olamarx) [inline index.html:26]: - [ ] Kindly maintain the design of the mobile version for the mobile version

## Your Design
image

## Figma
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @PrinceBadru, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!
s
## Highlights
- Linter passed ✔️ 
- Good use of Git and GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @PrinceBadru         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making all the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**